sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -102,15 +101,15 @@ export class IdentityClient extends ServiceClient implements INetworkModule {
       });
 
     if (response.bodyAsText && (response.status === 200 || response.status === 201)) {
-      const parsedBody = parse<{
-        token?: string;
-        access_token?: string;
-        refresh_token?: string;
-      }>(response.bodyAsText);
+      const parsedBody = JSON.parse(response.bodyAsText);
```

Comment:
You can still have your type information with

```typescript
JSON.parse(response.bodyAsText) as {
  token?: string;
  access_token?: string;
  refresh_token?: string;
};
```

Reply:
```ts
      const parsedBody: {
        token?: string;
        access_token?: string;
        refresh_token?: string;
      } = JSON.parse(response.bodyAsText);
```

this also works :)